.ci_support/linux_64_python3.12.____cpython.yaml

```diff
@@ -0,0 +1,42 @@
+MACOSX_SDK_VERSION:
+- '11.0'
+cdt_name:
+- cos6
+channel_sources:
+- conda-forge
+channel_targets:
+- conda-forge main
+cxx_compiler:
+- gxx
+cxx_compiler_version:
+- '12'
+docker_image:
+- quay.io/condaforge/linux-anvil-cos7-x86_64
+freetype:
+- '2'
+hdf5:
+- 1.14.2
+occt:
+- 7.7.2
+pcl:
+- 1.13.1
+pin_run_as_build:
+  occt:
+    max_pin: x.x
+  python:
+    min_pin: x.x
+    max_pin: x.x
+python:
+- 3.12.* *_cpython
+qt6_main:
+- '6.6'
+qt_main:
+- '5.15'
```

Comment:
could this be why both get installed?

Reply:
this wasn't a problem for pivy, so I guess this should not be the problem. Maybe one of the dependencies of freecad pulls qt-main.

Reply:
@conda-forge/core any ideas why qt-main and qt6-main are installed for the host-env?

Reply:
I think pivy pulls qt-main, we need pivy compiled with pyside6 for python 12

Reply:
I did the same with pivy here: https://github.com/conda-forge/pivy-feedstock/pull/44

Reply:
btw.: @adrianinsaval can you unpin coin in the freecad-bundle. so we get some users test if everything is ok with coin4.0.2 which should be available soon. I moved the problematic coin4.0.1 win build to broken, so it shouldn't be pulled anyways.

Reply:
thanks @adrianinsaval  coin3d 4.0.2 is uploaded.

Reply:
Good idea. We should use vtk-base in general